aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -1309,4 +1309,20 @@ module aptos_framework::account {
         account_state.guid_creation_num = MAX_GUID_CREATION_NUM - 1;
         create_guid(account);
     }
+
+    #[test_only]
+    struct FakeCoin { }
+
+    #[test(account = @0x1234)]
+    fun test_events(account: &signer) acquires Account {
+        let addr = signer::address_of(account);
+        create_account_unchecked(addr);
+        register_coin<FakeCoin>(addr);
+
+        let eventhandle = &borrow_global<Account>(addr).coin_register_events;
+        let counter = event::counter(eventhandle);
+        let event = CoinRegisterEvent { type_info: type_info::type_of<FakeCoin>() };
+        assert!(event::emitted_event(eventhandle, counter - 1) == option::some(event), 0);
+        assert!(event::emitted_event(eventhandle, counter) == option::none(), 0);
```

Comment:
Can you add in a test here that asserts the same return from the getter when it is called twice?

Reply:
e.g. 

```move
        let event = CoinRegisterEvent { type_info: type_info::type_of<FakeCoin>() };
        assert!(event::emitted_event(eventhandle, counter - 1) == option::some(event), 0);
        let event_2 = CoinRegisterEvent { type_info: type_info::type_of<FakeCoin>() };
        assert!(event::emitted_event(eventhandle, counter - 1) == option::some(event_2), 0);
```

Reply:
addressed